main/guides/getting-started/how-to-use-ui-kit.md

```diff
@@ -0,0 +1,147 @@
+# Introduction to Agoric UI Kit
+The Agoric UI Kit is a collection of components and tools designed to help developers build graphical user interfaces for Agoric decentralized applications (dapps). It provides a set of reusable React components, utilities for reading contract data from vstorage, functions for connecting to the user's wallet, and methods for executing offers on the Agoric blockchain.
+
+## Setup
+To use the Agoric UI Kit, you need to set up SES (Secure ECMAScript) in your application environment. SES is a secure runtime for JavaScript that ensures the security and integrity of your Agoric dapp. Refer to the [SES setup example](https://github.com/Agoric/dapp-inter/blob/main/src/main.tsx#L1) for instructions on enabling SES in your application. 
```

Comment:
I thought we were moving way from claiming "secure".

Reply:
No problem, removing "secure"